library/src/main/java/com/novoda/downloadmanager/lib/DownloadProvider.java

```diff
@@ -496,6 +307,12 @@ public String getType(final Uri uri) {
                     return mimeType;
                 }
             }
+            case BATCHES: {
```

Comment:
These weren't put in deliberately, if we want to keep it consistent then I'll remove them, sorry @xrigau :broken_heart: